_pkgdown.yml

```diff
@@ -65,18 +67,13 @@ reference:
     contents:
       - ex_adsl
       - DM
+  - title: Export
+    desc: Export into different object
+    contents:
+      - mpf_to_rtf
   - title: Miscellany
     desc: Other documented functions not interesting to end users
     contents:
       - print,ANY-method
       - ifnotlen0
-      - basic_matrix_form
```

Comment:
(Most of) these changes will need to be reverted because the functions should be documented

Reply:
Thanks @gmbecker , I am trying to familiar myself with the code base by trying to take some small issues at the moment :p